src/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py

```diff
@@ -333,6 +334,7 @@ def __pre_initialize(cls) -> list:
         cls._authManager = AuthManager(cls._getCSAuthorizarionSection(cls._fullComponentName))
 
         if not (urls := cls._pre_initialize()):
+            # cls.log is not yet available
```

Comment:
Maybe we should get rid of `sLog` in favour of something like this so `cls.log` always exists:
```python
class BaseRequestHandler(RequestHandler):
    # Subclasses should override this property
    log = gLogger.getSubLogger(__name__.split(".")[-1])
```

Reply:
Indeed, doing this way would avoid the very special case of `__pre_initialize`. 
But why should the subclass override it ?

Reply:
I should have said `This will be overridden in __initialize to be handler specific`.

Reply:
works with me, I change it now